download/community-packages.md

```diff
@@ -0,0 +1,22 @@
+# Community Packages
```

Comment:
Let's call it "Third-party" instead of "Community" because Kvrocks is also a community and can be ambiguous.

Reply:
done